### DISCUSSION TITLE
Warning popups for switches

### DISCUSSION THREAD

**Discussion by ghost** [home-assistant/frontend#11252] [Other feature requests]
## The request

Have a warning message to avoid accidentally turning off some important switches.
Example: Are you sure you want to turn off the switch?
## The alternatives

The alternative could be the long press of the switch-off button

**Comment by B-Kramer** [comment]
You can set the tap action--> confirmation option
https://www.home-assistant.io/lovelace/actions/#tap-action

**Comment by iantrich** [comment]
switches do not use tap_action, those would be for interacting with icon/name on the entity row
